templates/page.html

```diff
@@ -22,7 +22,7 @@
   <div id="header">
     <div class="header-container">
       <div class="header-left">
-        <a href="http://www.geoscience.gov.au/home"><img src="{{ url_for('static', filename='ggic-logo.png') }}" alt="AusGIN Logo" /></a>
+        <a href="http://www.geoscience.gov.au/home"><img src="{{ url_for('static', filename='AGSON-logo.png') }}" alt="ASGON Logo" /></a>
```

Comment:
@jwallace94 , I found a spelling mistake on a second glance -> alt="ASGON logo" instead of AGSON